lua/entities/gmod_wire_egp/lib/egplib/usefulfunctions.lua

```diff
@@ -235,6 +235,20 @@ else
 	end)
 end
 
+-- Used to check if the cached vertices of egpCircle etc are still valid, and if not update to the current values
```

Comment:
It would be cleaner and faster to handle this with a dirty flag that's set in `Obj.Receive` than by checking all the fields.

Reply:
I wasn't sure if the objects could be modified otherwis, ie loading saved frames.  
Although thinking about it now, that would also restore the cache correctly.